bundle/manifests/kueue-operator.clusterserviceversion.yaml

```diff
@@ -53,7 +53,7 @@ spec:
   customresourcedefinitions:
     owned:
     - kind: Kueue
-      name: kueues.operator.openshift.io
+      name: kueue.kueue.openshift.io
```

Comment:
kueues.kueue.openshift.io?

Reply:
I'm not really sure how this gets populated. This is generated from operator-sdk.